src/NzbDrone.Core/Notifications/NotificationService.cs

```diff
@@ -158,5 +169,31 @@ public void Handle(SeriesRenamedEvent message)
                 }
             }
         }
+
+        public void HandleAsync(HealthCheckCompleteEvent message)
```

Comment:
The actual update check should publish an event when an update is available which can be used to trigger the notification.